gluten-ui/src/main/scala/org/apache/spark/sql/execution/ui/GlutenSQLAppStatusListener.scala

```diff
@@ -52,7 +52,7 @@ private class GlutenSQLAppStatusListener(conf: SparkConf, kvstore: ElementTracki
         event.physicalPlanDescription,
         event.fallbackNodeToReason.toSeq.sortBy(_._1)
       )
-      kvstore.write(uiData)
+      kvstore.write(uiData, checkTriggers = true)
```

Comment:
I'm afraid it would introduce overhead for driver since each aqe stage would post one event. I think fire the trigger at  `onSQLExecutionStart` is enough.

Reply:
That's a good suggestion for me, thank you, fixed.